### DISCUSSION TITLE
Game Update 3028?

### DISCUSSION THREAD

**Discussion by pierrelasse** [YimMenu/YimMenu#2214] [Research]
So today i needed to update the game to version 3028 (atleast kiddions says it)
![image](https://github.com/YimMenu/YimMenu/assets/76223390/53dc09a0-fb34-4f94-802d-5c55e12471f8)

And i'm now wondering if anyone else received a update and if the menu is patched whatsoever.

Log when injecting the first time after the update
[cout.zip](https://github.com/YimMenu/YimMenu/files/12791123/cout.zip)

**Comment by NjuuOwO** [comment]
#2215 don't use it for now
